test/attachment-constraints.spec.js

```diff
@@ -75,12 +75,15 @@ describe('File attachment constraints:', function() {
             attachmentRefProp: 'bar.html' // The attachmentReference's maximum size of 40 overrides the document's maximum individual size of 25
           };
 
-          try {
-            testHelper.syncFunction(doc);
-            expect.fail('Expected maximum attachment violation exception not thrown');
```

Comment:
This line generates an exception that is subsequently caught by the `catch` block and then mishandled. Now it uses Chai's [throw](http://chaijs.com/api/bdd/) assertion to verify that the sync function threw an exception and that it was subsequently thrown by the `catch` block.